docs/sources/project/windows-client-test-server.md

```diff
@@ -0,0 +1,139 @@
+page_title: Set up a Windows server for client testing
+page_description: How to set up a server to test Docker Windows client
+page_keywords: development, inception, container, image Dockerfile, dependencies, Go, artifacts, windows
+
+
+# Windows Client test server
+
+In this section, you will learn how to setup a Windows machine for building and
+testing the Docker Windows client.
+
+Things you will need:
+
+- Windows Server/Desktop Machine
+- Java
+- MSysGit
+- TDM-GCC
+- MinGW (tar and xz)
+- Go with linux/amd64 cross-compiling
+- Docker.exe
```

Comment:
I mean people are more likely to run into problems like version mismatches etc. I don't think this step is necessary as they can get a docker.exe with `hack/make.sh binary` simply.